src/integrationTest/java/uk/gov/hmcts/darts/audio/service/AudioTransformationServiceHandleKedaInvocationForMediaRequestsTest.java

```diff
@@ -126,6 +127,14 @@ public void handleKedaInvocationForMediaRequestsShouldSucceedAndUpdateRequestSta
     @SuppressWarnings("PMD.LawOfDemeter")
     public void handleKedaInvocationForMediaRequestsShouldFailAndUpdateRequestStatusToFailedAndScheduleFailureNotificationFor(
         AudioRequestType audioRequestType) {
+
+        String  templateValues1 = """
+            {"hearing_date":"2023-11-30","start_time":"12:00:00","defendants":"","courthouse":"some-courthouse","end_time":"13:00:00","request_id":"1"}""";
+
+        String  templateValues2 = """
+            {"hearing_date":"2023-11-30","start_time":"12:00:00","defendants":"","courthouse":"some-courthouse","end_time":"13:00:00","request_id":"2"}""";
```

Comment:
maybe this one could test the scenario when defendants are populated?

Reply:
Agreed. I wanted to set up 2 tests one fail(populated) and pass(populated). Will update once done.